### PR TITLE
Fixed the null issue while parsing the ARM deployment output

### PR DESCRIPTION
**Pull request by pradudev**
**Task name**: AzureResourceGroupDeploymentV2 & AzureResourceManagerTemplateDeploymentV3

**Description**: Added a null check before parsing the ARM deployment output object

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) https://github.com/microsoft/azure-pipelines-tasks/issues/14246

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Review by bishal-pdMSFT — APPROVED** [review]

**Comment by pradudev** [comment]
@bishal-pdMSFT  thanks for reviewing my changes. Could you merge this changes in the master? Also, once merged, when can I expect this change in the latest version on AzuerDevOps ARM task?

**Comment by pradudev** [comment]
@bishal-pdMSFT @ddivanshu Hi, when can I expect the new version(v2.182.0 and v3.182.0) of the two ARM deployment tasks with the fixes in it?

**Pull request merged**
